app/jobs/integration_customers/create_job.rb

```diff
@@ -5,6 +5,7 @@ class CreateJob < ApplicationJob
     queue_as 'integrations'
 
     retry_on LagoHttpClient::HttpError, wait: :polynomially_longer, attempts: 3
+    retry_on BaseService::ThrottlingError, wait: :polynomially_longer, attempts: 100
```

Comment:
100 is a bit excessive. sidekiq uses 25 retries as a default, 25 seems more sensible 😄